pkg/machines/README.md

```diff
@@ -1,15 +1,26 @@
 # Cockpit VM Management
-TODO: Brief plugin desciption
+The 'Virtual Management' plugin provides basic overview of host virtualization by listing defined VMs along with their utilization and basic actions.
+
+The source of data is QEMU/Libvirt.   
 
 ## Nested Virtualization in Vagrant
-To play around with nested VMs, try when vagrant is up: 
+To play around with nested VMs in Vagrant on a host with enabled nested virtualization [1]:
+
+$ sudo vagrant up
 $ sudo vagrant ssh
```

Comment:
These lines need 4 spaces before each of them. Otherwise they're displayed as a single paragraph.